classes/submission/reviewAssignment/Collector.php

```diff
@@ -0,0 +1,298 @@
+<?php
+/**
+ * @file classes/submission/reviewAssignment/Collector.php
```

Comment:
This is very nice and clean and demonstrates very well the latest features we want to add throughout the codebase -- thanks!